ci/test/pipeline.template.yml

```diff
@@ -438,6 +438,17 @@ steps:
     agents:
       queue: linux-x86_64
 
+  - id: timelines
```

Comment:
I have been told before that this is relatively expensive and it's preferred to use an existing thingy. Does it make sense to use `restart` or `upgrade`?

Reply:
Good call, I moved them to `restart`.